### DISCUSSION TITLE
Kiali Multi-Cluster Feature

### DISCUSSION THREAD

**Discussion by Vineeth-varma** [kiali/kiali#4777] [Q&A]
Hi Team,

I just want to know does kiali supports Multi-cluster set Up, If yes can I get any relevant docs to go through it?

Thanks

**Comment by jmazzitelli** [comment]
see the multi-cluster docs here: https://kiali.io/docs/features/multi-cluster/

**Comment by Vineeth-varma** [comment]
I have gone through that docs, so it is like we just have to create two seperate kiali Instances for each primary istio-control plane. And do configuration changes needed for Kiali.
If I am wrong correct me.

**Comment by jmazzitelli** [comment]
converting to a Q&A discussion.

Yes, as of today you need one kiali per control plane.

There is ongoing discussions about how Kiali will support multi-cluster in the future. There are several threads in github issues and discussions that discuss these plans. For example: https://github.com/kiali/kiali/discussions/categories/multicluster

@lucasponce knows where the latest threads are in which multi-cluster is being discussed. Perhaps he can point you to the latest thread or issue/discussion?

**Reply by xu-jianchu** [reply]
Hi @jmazzitelli, is supporting Primary-Remote multi cluster deployment model somewhere on the Roadmap of Kiali?

**Comment by jshaughn** [comment]
Note that currently each Kiali needs a dedicated Prometheus, it does not work with a single prometheus unifying metrics from multiple control planes.  But, that may change soon, PRs for #4664  are in review.

**Comment by Xunzhuo** [comment]
Also there is a discussion I started https://github.com/kiali/kiali/discussions/4736 about this topic @Vineeth-varma
